docs/snippets/provision/alibaba.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: rdspostgresql
 spec:
   forProvider:
-    engine: postgresql
+    engine: PostgreSQL
```

Comment:
I _think_ either of these will work, but a new contributor had a successful run with existing today I believe

Reply:
I managed to come across some credentials and validated that `PostgreSQL` works. It's possible lowercase worked too, but either way we may as well be consistent between the two guides.